README.md

```diff
@@ -1695,6 +1695,7 @@ Other Style Guides
     + **Booleans** evaluate to **the value of the boolean**
     + **Numbers** evaluate to **false** if **+0, -0, or NaN**, otherwise **true**
     + **Strings** evaluate to **false** if an empty string `''`, otherwise **true**
+    + **NaN** comparisons always evaluate to false, even **NaN === NaN**
```

Comment:
This isn't actually true. This section is about *if statement* comparisons, so `NaN` is covered here under "**Numbers** evaluate to **false** if **+0, -0, or NaN**, otherwise **true**"

I'm fine including more information about `NaN !== NaN`, and `NaN` in general, but "if comparisons" aren't the section for it.